src/libPMacc/include/algorithms/math/doubleMath/exp.tpp

```diff
@@ -56,6 +56,17 @@ struct Log<double>
     }
 };
 
+template<>
+struct Log10<double>
+{
+    typedef double result;
+
+    HDINLINE double operator( )(const double& value )
```

Comment:
Space is missing after the parentheses. `HDINLINE double operator( )( const....)`